bay/images/Dockerfile.builder

```diff
@@ -1,4 +1,7 @@
-FROM uselagoon/php-7.4-cli-drupal:latest
+ARG PHP_VERSION=7.4
+FROM php:${PHP_VERSION}-cli-alpine AS php-cli
+FROM uselagoon/php-${PHP_VERSION}-cli-drupal:latest
+COPY --from=php-cli /usr/local/bin/phpdbg /usr/local/bin/
```

Comment:
Any reason we're not treating this as a dependency rather than an upstream copy?

```
RUN apk add --update --no-cache \
    php-phpdbg
```

Reply:
Well initially I naively thought that I couldn't add phpdbg using a pkg manager because it's a system API and not a PHP extension.

Turns out that's not the case but the complication to changing to your suggestion is that it installs the php 8 version of the module because the Alpine repos set in the build process don't use the edge branch. So implementing the suggestion would require that to change. 

It feels simpler to copy the binary but what's your preference?